<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>956</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Parallax</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>resources/sextant.png</normaloff>resources/sextant.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color: rgb(00,00,00);
color: #FFFFFF;
}
QPushButton{
	background-color: black;
}
 QPushButton:pressed {
     background-color: rgb(224, 0, 0);     
}
QPushButton:hover {
     background-color: rgb(100, 30, 30);
}
QPushButton#startButton:disabled:checked {
	color: gray;
}
QPushButton#startButton:disabled:checked {
	background-color: #ffaaaa;
}
QPushButton#startButton:disabled:!checked {
	background-color: lightGreen;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QMdiArea" name="mdiArea">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>400</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget{
background-color: rgb(11,11,11);
color: #FFFFFF;
}</string>
        </property>
        <property name="background">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>47</red>
           <green>47</green>
           <blue>47</blue>
          </color>
         </brush>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>956</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCameras">
    <property name="title">
     <string>Cameras</string>
    </property>
    <widget class="QMenu" name="menuDevices">
     <property name="title">
      <string>Devices</string>
     </property>
    </widget>
    <addaction name="menuDevices"/>
    <addaction name="separator"/>
    <addaction name="actionStreaming"/>
    <addaction name="separator"/>
    <addaction name="actionRecording"/>
    <addaction name="actionSnapshot"/>
    <addaction name="actionDir"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionDocumentation"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
    <addaction name="actionContact_Support"/>
   </widget>
   <widget class="QMenu" name="menuStages">
    <property name="title">
     <string>Stages</string>
    </property>
    <addaction name="actionServer"/>
    <addaction name="actionSave_Info"/>
   </widget>
   <addaction name="menuCameras"/>
   <addaction name="menuStages"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionStreaming"/>
   <addaction name="actionRecording"/>
   <addaction name="actionSnapshot"/>
   <addaction name="separator"/>
   <addaction name="actionDir"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QToolBar" name="toolBar_stages">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionServer"/>
   <addaction name="actionSave_Info"/>
  </widget>
  <action name="actionStreaming">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources/resources_from_qt.qrc">
     <normaloff>:/Icons/play.png</normaloff>
     <normalon>:/Icons/pause.png</normalon>:/Icons/play.png</iconset>
   </property>
   <property name="text">
    <string>Streaming</string>
   </property>
  </action>
  <action name="actionRecording">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources/resources_from_qt.qrc">
     <normaloff>:/Icons/recordingButton.png</normaloff>
     <normalon>:/Icons/recording_on.gif</normalon>:/Icons/recordingButton.png</iconset>
   </property>
   <property name="text">
    <string>Video Recording</string>
   </property>
  </action>
  <action name="actionSnapshot">
   <property name="icon">
    <iconset resource="resources/resources_from_qt.qrc">
     <normaloff>:/Icons/snapshotButton_white.png</normaloff>:/Icons/snapshotButton_white.png</iconset>
   </property>
   <property name="text">
    <string>Snapshot</string>
   </property>
  </action>
  <action name="actionDir">
   <property name="icon">
    <iconset resource="resources/resources_from_qt.qrc">
     <normaloff>:/Icons/folder.png</normaloff>:/Icons/folder.png</iconset>
   </property>
   <property name="text">
    <string>File Location</string>
   </property>
  </action>
  <action name="actionDocumentation">
   <property name="text">
    <string>Documentation</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionContact_Support">
   <property name="text">
    <string>Contact Support</string>
   </property>
  </action>
  <action name="actionServer">
   <property name="icon">
    <iconset resource="resources/resources_from_qt.qrc">
     <normaloff>:/Icons/mpmServer.png</normaloff>:/Icons/mpmServer.png</iconset>
   </property>
   <property name="text">
    <string>Server Config</string>
   </property>
  </action>
  <action name="actionSave_Info">
   <property name="icon">
    <iconset resource="resources/resources_from_qt.qrc">
     <normaloff>:/Icons/save.png</normaloff>:/Icons/save.png</iconset>
   </property>
   <property name="text">
    <string>Save Info</string>
   </property>
  </action>
  <action name="actionMock_Camera">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Mock Camera</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources/resources_from_qt.qrc"/>
 </resources>
 <connections/>
</ui>
